src/test/java/org/opensearch/sdk/TestSDKClient.java

```diff
@@ -41,4 +43,19 @@ public void testCreateClient() throws Exception {
         sdkClient.doCloseRestClient();
     }
 
+    @Test
+    public void testCreateHighLevelRestClient() throws Exception {
+        RestHighLevelClient testClient = sdkClient.initializeRestClient("localhost", 9200);
+
+        // Using the package name here as Java uses package name if the filename from different packages are same
```

Comment:
This comment is confusing.  We import this package/class specifically above.  What other `CreateIndexRequest` is potentially being used here?

Reply:
There are two `CreateIndexRequest`:
1. From Java Client: `org.opensearch.client.opensearch.indices.CreateIndexRequest`
2. From HLRC: `org.opensearch.client.indices.CreateIndexRequest`

I tried to look for different options to avoid importing it like this. Came across the below
1. Create a different test file to import same file name from different package
2. Use package name to let Java know which exact package we are referring to. Ref [here](https://softwareengineering.stackexchange.com/a/307631)

Open to suggestions here

Reply:
I don't see where `org.opensearch.client.opensearch.indices.CreateIndexRequest` is imported in this class.  Since there is a statement `import org.opensearch.client.opensearch.indices.CreateIndexRequest;` does that not specifically indicate the correct import?

Reply:
We can't import same filename from different package as it throws the error while importing
```
import org.opensearch.client.indices.CreateIndexRequest;
```
```
'org.opensearch.client.opensearch.indices.CreateIndexRequest' is already defined in a single-type import
```